tests/ops/models/test_application_config.py

```diff
@@ -83,3 +210,301 @@ def test_get_api_config(self, db: Session, example_config_record: Dict[str, Any]
         assert (
             ApplicationConfig.get_api_set_config(db) == example_config_record["api_set"]
         )
+
+    def test_set_config_set_config(self, db):
+        ApplicationConfig.set_config_set_config(db, CONFIG)
+        # check a few specific config properties of different types on the database record
+        db_config = ApplicationConfig.get_config_set_config(db)
+        assert (
+            db_config["notifications"]["notification_service_type"]
+            == CONFIG.notifications.notification_service_type
+        )
+        assert (
+            db_config["notifications"]["send_request_completion_notification"]
+            == CONFIG.notifications.send_request_completion_notification
+        )
+        assert (
+            db_config["execution"]["masking_strict"] == CONFIG.execution.masking_strict
+        )
+        assert db_config["redis"]["port"] == CONFIG.redis.port
+        assert db_config["security"]["cors_origins"] == CONFIG.security.cors_origins
+        # and perform a general check on the config objects
+        assert dumps(db_config) == CONFIG.json()
```

Comment:
I think if `assert dumps(db_config) == CONFIG.json()` would cover all the previous asserts right? So you could remove the other asserts and only have this one?

Reply:
ha, good call. that looks kinda silly in retrospect :)